gantry/simulator/CMakeLists.txt

```diff
@@ -1,18 +1,67 @@
+# Load free for the posix port
+find_package(FreeRTOS)
+
 # Set up the include directories
 include_directories(
         .
+        ../firmware
```

Comment:
Why do we need this? Ideally we can keep all of firmware completely separate from all of simulator. We should consider refactoring until we don't need this.

Reply:
For the freertos config